formbricks/formbricks
# fix: Preview for short urls

## What does this PR do?



Fixes #1819 



## How should this be tested?



- Create a short url 
- Check if preview is working

## Checklist



### Required


### Appreciated
